static/src/js/components/AccessMethod/AccessMethod.js

```diff
@@ -300,20 +474,36 @@ export class AccessMethod extends Component {
 
         case 'OPeNDAP': {
           id = `${collectionId}_access-method__opendap_${methodKey}`
-          title = 'Customize'
-          subtitle = 'OPeNDAP'
-          description = 'Select options like variables, transformations, and output formats for direct access via link or script.'
-          details = 'The requested data files will be made available for access immediately. Files will be accessed from a list of links in the browser or by using a download script.'
+          title = 'Customize with OPeNDAP'
+          description = 'Select options like variables, transformations, and output formats for direct access via link or script'
+          details = 'The data will be made available for access immediately.'
 
           break
         }
 
         case 'Harmony': {
           id = `${collectionId}_access-method__harmony_${methodKey}`
-          title = 'Customize'
-          subtitle = 'Harmony'
-          description = 'Select options like variables, transformations, and output formats for in-region cloud access.'
-          details = `The requested data will be processed using the ${name} service and stored in the cloud for analysis.`
+          title = name
+          description = descriptionFromMetadata
+          hasHarmony = true
+          hasSpatialSubsetting = hasShapefileSubsetting || hasBBoxSubsetting
+          hasFormats = supportedOutputFormats.length > 1
+          hasProjections = supportedOutputProjections.length > 1
+          /**
+           * CMR defines has-transforms as including interpolation or multiple projections, but
```

Comment:
Lets try to rephrase this as a TODO with some action like: `TODO: Add support for interpolation`